src/app/services/user.ts

```diff
@@ -8,6 +10,21 @@ class UserService implements IUserService {
     this.#userRepositories = repositories;
   }
 
+  login = async (username: string, password: string) => {
+    const user = await this.#userRepositories.getUserByLoginAndPassword(
+      username,
+      password
+    );
+
+    if (user) {
+      const token = makeToken(user);
```

Comment:
That is a very bad idea to save the user's password to the JWT token. At this point, you should pick only necessary fields to be saved to the token.